.github/workflows/publish.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish
+on:
+  release:
+    types:
+      - published
+jobs:
+  publish:
+    name: Publish to RubyGems
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+          ruby-version: '2.7'
```

Comment:
The example workflow at https://github.com/segiddins/rubygems-await/blob/main/.github/workflows/push_gem.yml has `ruby-version: ruby`, which is a value I cannot find documentation for.